bindata/network/ovn-kubernetes/common/ipsec.yaml

```diff
@@ -236,27 +236,28 @@ spec:
             cpu: 10m
             memory: 100Mi
         terminationMessagePolicy: FallbackToLogsOnError
-        readinessProbe:
+        livenessProbe:
           exec:
             command:
             - /bin/bash
             - -c
             - |
               #!/bin/bash
               ovs-appctl -t ovs-monitor-ipsec ipsec/status && ipsec status
-          initialDelaySeconds: 120
-          periodSeconds: 30
-          timeoutSeconds: 40
-        livenessProbe:
+          periodSeconds: 60
+          timeoutSeconds: 20
```

Comment:
A single probe timed out after 20 seconds (before, this was at the default value = 1 seconds, so this should be more resilient now)